pkgs/development/libraries/gcr/default.nix

```
@@ -32,6 +32,8 @@ stdenv.mkDerivation rec {
 
   outputs = [ "out" "dev" ];
 
+  configureFlags = [ "--with-libgcrypt-prefix=${stdenv.lib.getDev libgcrypt.dev}" ];
```

```suggestion
  configureFlags = [ "--with-libgcrypt-prefix=${stdenv.lib.getDev libgcrypt}" ];
```

```suggestion
  configureFlags = [ "--with-libgcrypt-prefix=${libgcrypt.dev}" ];
```

`.getDev` is to get .dev, isn't it ?